CMakeLists.txt

```diff
@@ -10,17 +10,18 @@ if(NOT CMAKE_BUILD_TYPE)
         FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
-set(HDF5_ECF_PLUGIN_BUILD_PATH "${CMAKE_BINARY_DIR}/lib/hdf5/plugin" CACHE INTERNAL "" FORCE)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib/hdf5/plugin)
```

Comment:
is this change required ? now the variable `HDF5_ECF_PLUGIN_BUILD_PATH` is undefined right ?
I don't remember the specifics, but I am not sure it's a drop-in replacement for what was done before (the `cmake -E copy [..] ${HDF5_ECF_PLUGIN_BUILD_PATH}`, especially for windows)

Reply:
This part was only to remove the copy of the library to the `HDF5_ECF_PLUGIN_BUILD_PATH` directory.  I can remove it if you think it's better to leave like it was before.

Reply:
I don't remember, but it may be necessary, especially for Windows.
Better leave it as it is, if it does not create problems for you.